extension-symbol-descriptor.md

```diff
@@ -1,6 +1,11 @@
-# workspace/xreferences extension to LSP
+# symbolDescriptor extensions to LSP
 
-The `workspace/xreferences` extension to the Language Server Protocol (LSP) enables a language server to export all of the references made from the workspace's code to its dependencies. Additionally, a new `textDocument/xdefinition` method performs the equivalent of `textDocument/definition` while returning metadata about the definition and _optionally_ a concrete location to it.
+Roughly speaking, a `symbolDescriptor` (defined fully below) encodes enough metadata about a symbol to be to uniquely identify it even when it appears in other projects (i.e. as a dependency).
+
+The `symbolDescriptor` extensions to the Language Server Protocol (LSP) allow a language server to able to interact with `symbolDescriptors` via the following methods:
+- The `textDocument/xdefinition` method is similar to `textDocument/definition` - it returns a `symbolDescriptor` and _optionally_ a concrete location for the definition
+- The `workspace/xreferences` method locates project wide references to a symbol, given a `symbolDescriptor`
+- The `workspace/symbol` method allows you to search for an instance of a symbol, given a `symbolDescriptor`
```

Comment:
I find "instance" to be confusing here, this searches for definitions of a symbol. Note that this is not the sole purpose of workspace/symbol, the method is part of the official spec. This extension just adds a filter parameter to it.